cnc_ctrl_v1/Axis.cpp

```diff
@@ -96,7 +89,7 @@ void   Axis::computePID(){
         return;
     }
     
-    _pidInput      =  motorGearboxEncoder.encoder.read()/_encoderSteps;
+    _pidInput      =  motorGearboxEncoder.encoder.read()/ *_encoderSteps;
```

Comment:
These pointers save us space, but they also ensure that if the value is changed in one place it will be changed everywhere, right? That's an improvement on two fronts!

Reply:
Yes, you are correct.  I still left the function call to changeEncoderResolution() in case we want any hooks to run on a change in the steps.